docs/src/css/custom.css

```diff
@@ -65,8 +65,7 @@
 
 
 [data-theme='dark'] .theme-doc-markdown a:not(.home-page a) {
-  color: white;
```

Comment:
You can revert this change. We want the link color to remain white (not blue) in dark mode for regular pages.

Only for the info boxes, we want the link color to be black instead of blue or white. So you can change the link color to back only for info boxes for both light and dark mode.